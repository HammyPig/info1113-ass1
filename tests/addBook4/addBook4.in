add book duplicates.csv 111111
add book nofile 111112
aDD BOOK duplicates.csv 111112
ADD BOOK duplicates.csv 111113
add book duplicates.csv 999999
LIST ALL
number copies