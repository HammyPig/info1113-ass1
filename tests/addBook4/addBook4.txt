adds books that can be found, cannot add books that can't be found