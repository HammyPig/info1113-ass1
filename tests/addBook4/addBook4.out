user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: No such file.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Iliad (Homer).

user: No such book in file.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)

user: The Iliad (Homer): 1
To Kill a Mockingbird (Harper Lee): 2

user: Ending Library process.
